sidecar/internal/service/reclaimspace.go

```diff
@@ -52,6 +54,40 @@ func (rs *ReclaimSpaceServer) RegisterService(server grpc.ServiceRegistrar) {
 	proto.RegisterReclaimSpaceServer(server, rs)
 }
 
+// accessModeStrToInt convert access mode type string to int32.
+// Make sure to update this function as and when there are new modes introduced.
+func accessModeStrToInt(mode v1.PersistentVolumeAccessMode) csi.VolumeCapability_AccessMode_Mode {
+	switch mode {
+	case v1.ReadWriteOnce:
+		return csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER
+	case v1.ReadOnlyMany:
+		return csi.VolumeCapability_AccessMode_MULTI_NODE_READER_ONLY
+	case v1.ReadWriteMany:
+		return csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER
+	case v1.ReadWriteOncePod:
+		return csi.VolumeCapability_AccessMode_SINGLE_NODE_SINGLE_WRITER
+	}
+
+	return csi.VolumeCapability_AccessMode_UNKNOWN
+}
+
+// getSecret get the secret details by name.
+func getSecret(kube *kubernetes.Clientset, ns, name string) (map[string]string, error) {
```

Comment:
kube *kubernetes.Clientset to `client *kubernetes.Clientset`

Reply:
use `kubeClient` as standard throughout the project ?? 
just client will refer to grpc client pointing towards the csi driver.

Please move internal functions to the end of file after exported/implemented functions for readability(atleast what i personally follow :sweat_smile: )